### ISSUE TITLE
Menu - view more does not announce to screen reader focus change

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5840]
**Describe the issue. What is the expected and unexpected behavior?**
When using the view more example, focus correctly changes to the next menu item, however it doesn't announce to a screen reader that the focus has changed. I played with an `aria-activedescendant` approach, however in this investigation I realized that with this approach focus would need to be handled differently (i.e. for an `aria-activedescendant` approach, focus would need to stay on the parent menu and then the attributes would communicate what menu item has focus, visually it would probably look more like the Typeahead Select). We should take a look at what ways we can use to communicate this change of focus. `aria-activedescendant` was an idea, but maybe there's an alternative that we can find to fix this issue.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Use VO and shift focus through the menu items, select the view more button, afterwards VO announces "end of list" instead of shifting focus to next item.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug

**Comment by tlabaj** [comment]
@mcarrano can we please add this to a milestone
